### ISSUE TITLE
make lib wrong

### ISSUE THREAD

**Issue by 10183308** [xieyufei1993/InceptText-Tensorflow#5]
the info is bellow

4 errors detected in the compilation of "/tmp/tmpxft_0000762e_00000000-6_deform_psroi_pooling_op_gpu.cu.cpp1.ii".

could you help me?

**Comment by FakerYFX** [comment]
you can follw this one:https://github.com/Zardinality/TF_Deformable_Net,just need to run the lib/make.sh

**Comment by ghost** [comment]
@10183308   I had teh same error ,How do I solve this problem?  Thank you .
@xieyufei1993

**Comment by ghost** [comment]
tensorflow's version ==1.4.1

**Comment by FakerYFX** [comment]
follow this One https://github.com/Zardinality/TF_Deformable_Net

**Comment by ghost** [comment]
@xieyufei1993 
[https://github.com/Zardinality/TF_Deformable_Net/issues/20]
10 errors detected in the compilation of "/tmp/tmpxft_00003e41_00000000-7_deform_conv.cu.cpp1.ii".
请问你是怎么解决的？我还是报这个错误，实在搞不懂。

**Comment by FakerYFX** [comment]
what is your gcc version and python version?

**Comment by 10183308** [comment]
我也是遇到这个问题，没有解决掉。

**Comment by ghost** [comment]
@xieyufei1993  
cuda 8.0  cuDNN 5.x
gcc version 4.9.2 20150212 (Red Hat 4.9.2-6) (GCC) 
gcc version 5.3.1 20160406 (Red Hat 5.3.1-6) (GCC) 
**I am learning your code, but I am getting an error during compilation.**
line 64  in **make.sh**:
nvcc -std=c++11 -c -o deform_psroi_pooling_op.cu.o deform_psroi_pooling_op_gpu.cu.cc \
-I $TF_INC -D GOOGLE_CUDA=1 -x cu -Xcompiler -fPIC -D GOOGLE_CUDA -arch=$ARCH
**If remove** -D GOOGLE_CUDA=1 , the compilation can pass,But get errors when running , the file could not be found.
can't find deform_psroi_pooling_op.cu.o
undefined symbol: _ZTIN10tensorflow8OpKernelE
**In short, the 64 lines are wrong, I don't know how to change.**

**Comment by FakerYFX** [comment]
just follow this one https://github.com/CharlesShang/TFFRCNN/issues/108

**Comment by ghost** [comment]
@xieyufei1993   
Thank you for your advice, I have solved the previous problem ,mainly the latter error :
**4 errors detected in the compilation of "/tmp/tmpxft_00005a43_00000000-7_deform_psroi_pooling_op_gpu.cu.cpp1.ii".**
g++: error: deform_psroi_pooling_op.cu.o: No such file or directory

**Comment by lqyiii** [comment]
@ccnankai @10183308  have you solved this problem? i also meet this error...

**Comment by MrXieJ** [comment]
@ccnankai  it bother me serveral days ,i finally solved it by myself ,only need to modify the make.sh file .have you solved the problem, i,d love to share my solution

**Comment by lqyiii** [comment]
i added this command (--expt-relaxed-constexpr) after which line has nvcc.  the error disappear.
eg,
nvcc -std=c++11 -c -o roi_pooling_op.cu.o roi_pooling_op_gpu.cu.cc \
	-I $TF_INC -I $NSYNC_INC -D GOOGLE_CUDA=1 -L $CUDA_HOME/lib64 -x cu -Xcompiler -fPIC -D GOOGLE_CUDA -arch=$ARCH --expt-relaxed-constexpr

**Comment by aditipanda** [comment]
> i added this command (--expt-relaxed-constexpr) after which line has nvcc. the error disappear.
> eg,
> nvcc -std=c++11 -c -o roi_pooling_op.cu.o roi_pooling_op_gpu.cu.cc
> -I $TF_INC -I $NSYNC_INC -D GOOGLE_CUDA=1 -L $CUDA_HOME/lib64 -x cu -Xcompiler -fPIC -D GOOGLE_CUDA -arch=$ARCH --expt-relaxed-constexpr

@lqyiii  Even after adding this command (--expt-relaxed-constexpr), at the end of every nvcc, I am still getting the following error:

![screenshot from 2018-11-14 11-49-11](https://user-images.githubusercontent.com/25290812/48464061-226ecc80-e804-11e8-9853-c01074dba967.png)

**Comment by aditipanda** [comment]
@MrXieJ  what changes did you do in make.sh?

**Comment by jiangzz1628** [comment]
各位大佬问题都解决了吗，我的也是没有那些个.o文件 @10183308 @xieyufei1993 @ccnankai @lqyiii 
按照各位的解决方案试了，也不行
